networkx/generators/tests/test_geometric.py

```diff
@@ -329,3 +329,47 @@ def test_geometric_edges_raises_no_pos():
     msg = "All nodes in `G` must have a 'pos' attribute"
     with pytest.raises(nx.NetworkXError, match=msg):
         nx.geometric_edges(G, radius=1)
+
+
+class TestS1Graph:
```

Comment:
No need to create a class structure here :)

I know we have classes all across our tests but we are trying to move away from them.

Reply:
I removed the class there